entries/t/telstra.com.au.json

```diff
@@ -1,16 +1,13 @@
 {
   "Telstra": {
     "domain": "telstra.com.au",
-    "url": "https://www.telstra.com.au/",
-    "contact": {
-      "facebook": "Telstra24x7",
-      "twitter": "Telstra"
-    },
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "documentation": "https://www.telstra.com.au/mytelstra/my-telstra-security",
     "keywords": [
       "utilities"
-    ],
-    "regions": [
-      "au"
     ]
```

Comment:
Is it no longer just for Australian customers?

Reply:
It is. I didn't know that field was to be kept.